src/routes/grants/index.svelte

```diff
@@ -0,0 +1,134 @@
+<script context="module">
+  export async function preload() {
+    const posts = await this.fetch(`blog.json`)
+      .then((r) => r.json())
+      .then((posts) => posts);
+    const events = await this.fetch(`events.json`)
+      .then((r) => r.json())
+      .then((events) => events);
+    return {
+      posts,
+      events,
+    };
+  }
+
+  import memosList from "../../../content/grants/memos";
```

Comment:
Fetch memos and projects in the preload function.

Reply:
Solved in `commit 4fa7acc07bc31c9d5715489c2a857ce5c11cd318`